src/Kunstmaan/AdminBundle/Resources/config/services.yml

```diff
@@ -57,6 +56,12 @@ services:
         tags:
             -  { name: 'twig.extension' }
 
+    kunstmaan_admin.multidomain.twig.extension:
```

Comment:
We can not put this in the MultiDomainBundle because the twig function needs to be available also when the MultiDomainBundle is not enabled.